index.html

```diff
@@ -1,29 +1,69 @@
     <script defer src="./js/artists.js"></script>
+    <script defer src="./js/popup.js"></script>
+    <script defer src="./js/navigation.js"></script>
+    <script src="https://kit.fontawesome.com/76eab5b41f.js" crossorigin="anonymous"></script>
 </head>
+
 <body>
+    <header>
+        <div id="header-1">
+            <ul>
+                <li><i class="fa-brands fa-facebook-f"></i></li>
+                <li><i class="fa-brands fa-twitter"></i></li>
+                <li>English</li>
+                <li>My page</li>
+                <li>Logout</li>
+            </ul>
+        </div>
+        <div id="header-2">
+            <aside id="popup">
+                <ul id="popup-list">
+                    <li>About</li>
+                    <li>Program</li>
+                    <li>Join</li>
+                    <li>Sponsor</li>
+                    <li>News</li>
+                    <li>Home</li>
+                </ul>
+            </aside>
+            <i class="fa-solid fa-bars" id="bars-button"></i>
+            <img src="./images/logo.jpg" id="header-img" />
+            <ul id='desktop-menu'>
+                <li>About</li>
+                <li>Program</li>
+                <li>Join</li>
+                <li>Sponsor</li>
+                <li>News</li>
+                <li>Home</li>
+            </ul>
+        </div>
+    </header>
+
     <main>
-        <div id="header-text">
-            <h1 class="title bg">ALL POINTS<br />
+        <div id="header-wrapper">
+            <img id="stage" src="./images/stage.jpg" alt="Stage photo" />
+            <h1 id="title">ALL POINTS<br />
                 EAST FEST<br />
                 2023</h1>
-        </div>
-        <p id="event-desc">
-            All Points East is an annual music festival held over two weekends in
-            London's Victoria Park, run by AEG Presents. The 10-day event comprises
-            one festival weekend, four days of free entry and community activities
-            known as "In the Neighbourhood" and finally three standalone headline
-            shows in their APE Presents series. 
-        </p>
-        <div id="date-address">
-            <p id="date">2023.08.18(FRI) ~ 28(MON)</p>
-            <address>@ Victoria Park, London</address>
+            <p id="event-desc">
+                All Points East is an annual music festival held over two weekends in
+                London's Victoria Park, run by AEG Presents. The 10-day event comprises
+                one festival weekend, four days of free entry and community activities
+                known as "In the Neighbourhood" and finally three standalone headline
+                shows in their APE Presents series.
+            </p>
+            <div id="date-address">
+                <p id="date">2023.08.18(FRI) ~ 28(MON)</p>
+                <address>@ Victoria Park, London</address>
+            </div>
         </div>
         <section id="program-activities">
             <div id="activities-header">
```

Comment:
- [ ] Same thing goes for this section too. Kindly add logo  to each of the program card and increase the width of the card so as to match how it appears on the template. You will also need to pay attention the font-size on your design.

### Your Design
![your-main-program](https://github.com/Movgto/capstone_1_project/assets/26597432/9c184f45-58f9-4377-94b3-be5bd545c09a)

### Template
![temp-main-program](https://github.com/Movgto/capstone_1_project/assets/26597432/bce3fb28-e094-4728-af55-6cd4fad00e9f)


- [ ] Same thing goes for the mobile. Kindly walk your design alongside the template given

### Your Design
![your-main-program-mobile](https://github.com/Movgto/capstone_1_project/assets/26597432/1fa2e058-9826-4e72-9666-dc151b07d9d3)

### Template
![temp-main-pgrogram-mobile](https://github.com/Movgto/capstone_1_project/assets/26597432/6be7154b-8690-4467-ad57-0e9a4e869d12)

Reply:
- [ OPTIONAL] I just noticed that you added Icons on each program but I can see that your Title and your descriptions are not well design according the design given on this project ,also put your description in lowercase as we can see on the design check this 
![Screenshot from 2023-07-06 22-03-37](https://github.com/Movgto/capstone_1_project/assets/88894525/2b131bed-8b66-4bcc-938a-361038772290)
 
> ### Template
> ![temp-main-pgrogram-mobile](https://user-images.githubusercontent.com/26597432/251288025-6be7154b-8690-4467-ad57-0e9a4e869d12.png)

_I suggest you reduced the font-size of your Icons, so that it can give some space to your texts_